srml/aura/src/lib.rs

```diff
@@ -265,42 +222,15 @@ impl<T: Trait> Module<T> {
 		assert!(!slot_duration.is_zero(), "Aura slot duration cannot be zero.");
 
 		let last_slot = last / slot_duration.clone();
-		let first_skipped = last_slot.clone() + One::one();
 		let cur_slot = now / slot_duration;
 
 		assert!(last_slot < cur_slot, "Only one block may be authored per slot.");
-		if cur_slot == first_skipped { return }
-
-		let skipped_slots = cur_slot - last_slot - One::one();
-
-		H::handle_report(AuraReport {
```

Comment:
Note that AuRa-specific reporting/slashing has now been removed. One can use `im-online` together with `AuRa` to slash offline validators.

Reply:
since this slashing code is strictly better than im_online at detecting "malicious" offlineness, then it should be reintegrated properly in a follow-up PR. maybe make an issue for that.